### PR TITLE
(blog): How Impossible Foods Used Gatsby + Airtable to Build A Store Locator Map

### PR DESCRIPTION
**Pull request by MaedahBatool**
Work in progress…

Preview: https://build-9c78b597-dd5b-4d32-b124-f953ac87d6bc.gtsb.io/blog/2020-02-11-how-impossible-foods-used-gatsby-airtable-to-build-a-store-locator-map/

### CODE REVIEW

**Comment by MaedahBatool** [comment]
@hashimwarren since this is my first blog post PR can you please check if everything is alright so I can merge this PR. It's ready for review.

**Comment by wardpeet** [comment]
@MaedahBatool I didn't go over the content yet but is the author correct? It's authored as Sam and not you. Might be intentional :smile:

Here is a live preview https://build-9c78b597-dd5b-4d32-b124-f953ac87d6bc.gtsb.io/blog/2020-02-11-how-impossible-foods-used-gatsby-airtable-to-build-a-store-locator-map/

**Comment by MaedahBatool** [comment]
Hey, @wardpeet! 

Yes, the author is correct. Sam crafted this piece and I just made the final edits like formatting, proofreading, adding media files, etc. So, basically he's the main writer. Wonder if multiple authors are allowed? 🤔

**Comment by MaedahBatool** [comment]
@AishaBlake can you please review this PR since Hashim is away. He has asked me to add you as a reviewer. Thank you!

File: docs/blog/2020-02-11-how-impossible-foods-used-gatsby-airtable-to-build-a-store-locator-map/index.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+2019 saw a flurry of launches:
+
+- A **food service hub** to support restaurants serving the product
+- Improvements to the help center managed by the customer service team
+- A new **Careers** page
+- An incredibly fast **turnaround for campaign pages** around Impossible Food’s partnerships with Burger King and Little Caesars.
+- Partnerships that were widely covered by media and rocketed Impossible Foods’ brand outreach into millions of households across the US.
+
+![Results](./results.jpg)
```
Comment (AishaBlake) [inline docs/blog/2020-02-11-how-impossible-foods-used-gatsby-airtable-to-build-a-store-locator-map/index.md]: Can we block the cursor out of this image? Should be a quick fix!

File: docs/blog/2020-02-11-how-impossible-foods-used-gatsby-airtable-to-build-a-store-locator-map/index.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+During this time, they found that their website was failing to help excited prospective customers find an Impossible Burger at a location near them. They showed users a static list of all Impossible Food locations, grouped by state.
+
+While this worked at some level, it was difficult for users to find store locations nearby them because they had to scan a long and growing list of locations, sorted alphabetically rather geographically, and then use other tools like Google Maps to figure out their closest location. Many users simply gave up rather than going through this complicated process, causing Impossible Foods to lose excited potential customers.
+
+<Pullquote>
+“Our product would be in certain places, and people wouldn't know. It wouldn’t be in other places, and people would guess that it was there” -- Ashley Geo, Head of Brand Marketing Communications, Impossible Foods.
+</Pullquote>
+
+![Old Maps](./old-maps.gif)
```
Comment (AishaBlake) [inline docs/blog/2020-02-11-how-impossible-foods-used-gatsby-airtable-to-build-a-store-locator-map/index.md]: I think this and `new-maps.gif` below move too quickly. I could see that movement bothering some users and, to my knowledge, there's no way to turn it off on the fly. Can we edit these?

**Pull request closed without merging**